client/VCMI_client.vcxproj.filters

```diff
@@ -22,6 +22,9 @@
     <ClCompile Include="windows\CCreatureWindow.cpp">
       <Filter>windows</Filter>
     </ClCompile>
+    <ClCompile Include="windows\CHeroOverview.cpp">
+      <Filter>windows</Filter>
+    </ClCompile>

```

Comment:
I don't think that anybody uses or updates these project files.

Should probably be removed completely to be honest.

Reply:
I add it usually to every file I find. :)

Yes. But out of scope here.